macros/sql/unpivot.sql

```diff
@@ -12,11 +12,11 @@ Arguments:
     value_name: Destination table column name for the pivoted values
 #}
 
-{% macro unpivot(relation=none, cast_to='varchar', exclude=none, remove=none, field_name='field_name', value_name='value') -%}
-    {{ return(adapter.dispatch('unpivot', 'dbt_utils')(relation, cast_to, exclude, remove, field_name, value_name)) }}
+{% macro unpivot(relation=none, cast_to='varchar', exclude=none, remove=none, field_name='field_name', value_name='value', quote_identifiers=False) -%}
```

Comment:
In an ideal world, I would rather have the default be `quote_identifiers=True` so that it matches with `pivot`. But agree with your choice here to default it to `False` instead so we don't have any breaking behavior change.